easybuild/tools/job/pbs_python.py

```diff
@@ -54,7 +54,7 @@
     from PBSQuery import PBSQuery
     KNOWN_HOLD_TYPES = [pbs.USER_HOLD, pbs.OTHER_HOLD, pbs.SYSTEM_HOLD]
 
-except ImportError as err:
+except ImportError:
```

Comment:
Same here, please restore and include error message in log message

Reply:
Automatic change w/o behavior change. Can be done later